include/mbedtls/mbedtls_config.h

```diff
@@ -962,410 +302,38 @@
 #define MBEDTLS_ERROR_STRERROR_DUMMY
 
 /**
- * \def MBEDTLS_GENPRIME
- *
- * Enable the prime-number generation code.
- *
- * Requires: MBEDTLS_BIGNUM_C
- */
-#define MBEDTLS_GENPRIME
-
-/**
- * \def MBEDTLS_FS_IO
+ * \def MBEDTLS_SELF_TEST
```

Comment:
Should be moved to crypto_config.h section "General and test configuration options". Same for MBEDTLS_TEST_CONSTANT_FLOW_MEMSAN, MBEDTLS_TEST_CONSTANT_FLOW_VALGRIND and MBEDTLS_TEST_HOOKS below.